testing/SetupTestbed.ps1

```diff
@@ -366,13 +380,63 @@ it is likely that the DNS entry was still successfully added. To
 verify, log on to DC1 and run 'Resolve-DnsName ls1' in PowerShell.
 If it returns NXDOMAIN, you'll need to add it manually."
 Write-Output "The time is $(Get-Date)."
-az vm run-command create `
+
+# Define the PowerShell script with the DomainName variable interpolated
+$scriptContent = @"

```

Comment:
Since you added a timeout for this (nice improvement by the way), I recommend you delete the old warning about the call hanging indefinitely.